Asterix/utils/gaussians.py

```diff
@@ -0,0 +1,85 @@
+
+import numpy as np
+import scipy.optimize as opt
+
```

Comment:
Remember that PEP8 asks for two blank lines before a function definition (not class methods though)>

Reply:
ok